lib/luadata.c

```diff
@@ -27,11 +27,17 @@ static int luadata_lnew(lua_State *L);
 
 LUNATIK_PRIVATECHECKER(luadata_check, luadata_t *);
 
+#define luadata_checkarg(L, arg, size, offset, length)					\
```

Comment:
I believe this could be an inline, right? I'm trying to use macros just in the case it can't be implemented as inline or it's a simple expression..